src/pages/en/guides/markdown-content.md

```diff
@@ -395,9 +417,14 @@ We also suggest [diving into their theme documentation](https://github.com/shiki
 #### Prism configuration
 
 When using Prism, you'll need to add a stylesheet to your project for syntax highlighting. If you're just getting started and prefer to use Prism over Shiki, we suggest:
+
 1. [Setting `syntaxHighlight: 'prism'`](#choose-a-syntax-highlighter) from your `@astrojs/markdown-remark` config.
 1. Choosing a premade stylesheet from the available [Prism Themes](https://github.com/PrismJS/prism-themes).
-2. Adding this stylesheet to [your project's `public/` directory](https://docs.astro.build/en/core-concepts/project-structure/#public).
-3. Loading this [into your page's `<head>`](https://docs.astro.build/en/core-concepts/astro-pages/#page-html) via a `<link>` tag.
+1. Adding this stylesheet to [your project's `public/` directory](https://docs.astro.build/en/core-concepts/project-structure/#public).
+1. Loading this [into your page's `<head>`](https://docs.astro.build/en/core-concepts/astro-pages/#page-html) via a `<link>` tag.
 
 You can also visit the [list of languages supported by Prism](https://prismjs.com/#supported-languages) for options and usage.
+
+## Notes
```

Comment:
Just checking: a header at this level is showing up in the right hand sidebar, so is this the best description to use? Given that this seems to be just one note, can the header be more descriptive? e.g "Heading IDs"?

And then, I don't think it's necessary to use a bullet point for this. I don't think it benefits from the point and the indentation.

Reply:
Since I didn't know where to put this information, I thought of a misc section. Maybe "Technical Details" would be a better name

Reply:
I've added a note about `<Markdown />` and SSR. What do you think? Do you like this approach or should I search another place to put them?

Reply:
Fred has an entire new PR #411 with warnings splashed all over the page about the Markdown component! So, I don't think we need it here.  😄  Take a look at that one, then probably I think just remove your SSR bit from here. Once THIS pr is merged, we can merge Fred's.